internal/security/bootstrapper/main.go

```diff
@@ -39,6 +39,7 @@ import (
 const (
 	securityBootstrapperServiceKey  = "edgex-security-bootstrapper"
 	configureDatabaseSubcommandName = "configureRedis"
+	configureKongSubCommandName     = "configureKong"
```

Comment:
i think we don't need this, can be removed

Reply:
Correct - left over code. That has been removed. Thanks!